### PR TITLE
English to GalCom

### PR DESCRIPTION
**Pull request by samo-novak**
Changed old references to English language on the Skrell page - replaced by Galactic Common.

### CODE REVIEW

File: The Skrell.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ ____
 
 **Naming**
 
-Skrell names either consist of chosen human names or rough translations, due to their language being something humans cannot vocalize. The translations appear to vary widly when converted to english, so there is a wide variety of names possible.
+Skrell names either consist of chosen human names or rough translations, due to their language being something humans cannot vocalize. The translations appear to vary widly when converted to the Galactic Common, so there is a wide variety of names possible.
```
Comment (GinjaNinja32) [inline The Skrell.md]: "the" Galactic Common?

Comment (samo-novak) [inline The Skrell.md]: I'm not sure if that's a problem, but I can remove it.
I'm not an English native speaker, and while I think I can speak English pretty well, there are still some details I don't know about.

Comment (GinjaNinja32) [inline The Skrell.md]: It'd be like talking about "the English"; except in English's case, "the English" denotes a people, not a language.

Comment (samo-novak) [inline The Skrell.md]: Oh, right. I didn't think about that. Thanks! ;)

**Pull request merged**